whitepapers/0003_token_bridge.md

```diff
@@ -236,10 +236,16 @@ NewContract [32]uint8
 
 ## Caveats
 
-There is no guarantee for completion of transfers. If a user initiates a transfer and doesn't call `completeTransfer`
-on the target chain, a transfer might not be completed. In case a guardian set change happens in-between and the
-original signer guardian set expires, the transfer will be stuck indefinitely.
+### Transfer completion
+A user who initiated a transfer should call `completeTransfer` within 24 hours. Guardian Sets are guaranteed to be valid for at least 24 hours. If the user waits longer, it could be that the Guardian Set has changed between the time where the transfer was initiated and the the time the user attempts to redeem the VAA. Let's call the Guardian Set at the time of signing `setA` and the Guardian Set at the time of redeeming on the target chain `setB`.
 
+If `setA != setB` and more than 24 hours have passed, there are multiple options for still redeeming the VAA on the target chain:
+1. The Guardians that signed the VAA may still be part of `setB`. In this case, the VAA meerly needs to be modified to have the new Guardian Set Index and it will be a valid VAA that can be redeemed.
+2. The intersection between `setA` and `setB` is greater than 2/3 of `setB`, but not all signatures of the VAA are from Guardians in `setB`. Then it may be possible to gather signatures from the other Guardians from historic gossip network observations, and replace now-invalid signatures with those.
+3. Guardians may elect to do the above automatically and make those updated VAAs available for users over the public RPC.
```

Comment:
What exactly does this mean? All VAAs are available via public RPC. This is how front-ends can tell a VAA is available to be redeemed. I don't think anyone currently does this automatically so this entry should be removed.
```suggestion
```

Reply:
Yeah I guess so. It just seems a bit hand wavy and superfluous. I like keeping that kind of stuff out of design docs, but your call.

Reply:
Removed this point since it was decided to not implement this feature